requirements.txt

```diff
@@ -0,0 +1,6 @@
+virtualenv
```

Comment:
If that's the case, instead of copying the requirements file, you could specify the following in the requirements.txt in the root.
```
virtualenv
-r modules/requirements.txt
```
This would include the requirements.txt from the modules directory